axe.d.ts

```diff
@@ -146,6 +153,14 @@ declare namespace axe {
 		}[];
 		cleanup?(callback: Function): void;
 	}
+	interface VirtualNode {
+		children?: VirtualNode[];
+		parent?: VirtualNode | null;
```

Comment:
This is saying `parent` is either `undefined`, `null` or a `VirtualNode`. IMO it should only be `null` or `VirtualNode`, never `undefined`.